programming/sublime-text/files/sublime_text.desktop

```diff
@@ -0,0 +1,23 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Sublime Text
+GenericName=Text Editor
+Comment=Sophisticated text editor for code, markup and prose
+Exec=/opt/sublime_text/sublime_text %F
+Terminal=false
+MimeType=text/plain;
+Icon=sublime-text
+Categories=TextEditor;Development;Programming;
+StartupNotify=true
+Actions=new-window;new-file;
+
+[Desktop Action new-window]
+Name=New Window
+Exec=/opt/sublime_text/sublime_text --launch-or-new-window
+OnlyShowIn=Unity;
+
+[Desktop Action new-file]
+Name=New File
+Exec=/opt/sublime_text/sublime_text --command new_file
+OnlyShowIn=Unity;
```

Comment:
As far as I can tell you seem to have an exact copy of the desktop file that is also included in the archive here.
Any particular reason for not simply using the latter?

Reply:
The old desktop file had `Name = Sublime Text 3` and `Exec=/opt/sublime_text_3/sublime_text_3 %F` etc. This now is the desktop file that sublime text now provides with their tarball. I originally tried to just copy from extracted archive in the script itself but that didn't work, so I bundled it. This should now never be needed to changed unless sublime-text company pivots again.

Reply:
Yes, I just meant to copy the .desktop file from the tarball instead of maintaining a separate file here in the repository.

Did you try a simple:

```
pisitools.insinto("/usr/share/applications/", "sublime_text/sublime_text.desktop") 
```
?

Not at my Solus computer atm so can't test unfortunately.